### ISSUE TITLE
rowHeight for Setup Options

### ISSUE THREAD

**Issue by chumaltd** [olifolkerd/tabulator#3149]
Hi, I'd like to have a option for setting cell height statically through row config.

Now, I have some rendering issue, relating to cell size auto calculation.
On every data, the first 20 rows are just fine, but following rows have large cells in width & height.

I use Tabulator v4.9.3 with Preact, Redux and so on.
On the other hand, supplying table data from parsed JSON works well at all...
I confirmed Redux data is healthy equal as JSON.

Anyway, static cell size can render proper tables even in this case.
Width can be controlled with `width` setup option for each column.
And, following rowFormatter can fix height.

```
rowFormatter: (row)=>{
    row.getCells().forEach((cell) => {
        cell._cell.height = '30px';
    });
} 
```

I think we may see other auto sizing issues, and static size option can be a safeguard.
For simple table, static row height will be enough.

**Comment by olifolkerd** [comment]
Hey @chumaltd 

I have just pushed an update to the 5.2 branch to add the new `rowHeight` option

This will be included in the 5.2 release later this month.

Cheers

Oli :)

**Comment by chumaltd** [comment]
@olifolkerd , Happy to hear that!
My product is still running on v4.9.
So I'll migrate to 5.x on the next try, and confirm the rendering behavior with preact.
Thank you for your great work!

**Comment by chumaltd** [comment]
Hi @olifolkerd , I'm just working on v5.2, and getting things mostly successful.

On `rowHeight`, it works almost well.
But for my case, it renders white margin under data.

I found out the situation:

* "progress" formatter sets `height: 100%`, and then rows have extremely large heights
* `rowHeight` option adjusts row heights properly. But, total table height looks still `100%` accumulated height
* Therefore, difference between `rowHeight` data and the total table height remains white space

And I just solved my case.

If I have `<style>.tabulator-cell { max-height: 30px }</style>` with `rowHeight: 30`, table height is also calculated properly.

This is much cleaner than v4.9.
I think it's enough.

I don't know other than "progress" case, but CSS `max-height` for cells can work several cases.

Thanks!
